pycheribuild/mtree.py

```diff
@@ -255,6 +256,29 @@ def __contains__(self, item):
         mtree_path = self._ensure_mtree_path_fmt(str(item))
         return mtree_path in self._mtree
 
+    def filter(self, globs, exceptions=[], print_status=False):
```

Comment:
```suggestion
    def filter_out(self, globs, exceptions=[], print_status=False):
```
? This to me implies positive not negative filtering.

Reply:
Thinking this morning, I wonder if `exclude_matching` would be better (it had been `filter_matching` initially in my tree)